cruft/_commands/utils/generate.py

```diff
@@ -36,6 +36,8 @@ def cookiecutter_template(
 
     repo.head.reset(commit=commit, working_tree=True)
 
+    if repo.working_dir is None:
```

Comment:
When would this

Reply:
Never, it is to appease mypy

Reply:
Can we do an assert then? That conveys the intention better (asserting the internal state is consistent) rather than checking and raising a runtime error. That should also fix mypy issues.